libcalico-go/config/crd/crd.projectcalico.org_felixconfigurations.yaml

```diff
@@ -118,6 +118,26 @@ spec:
                   logs are emitted to the BPF trace pipe, accessible with the command
                   `tc exec bpf debug`. [Default: Off].'
                 type: string
+              bpfMapSizeConntrack:
+                description: BPFMapSizeConntrack sets the size for conntrack map
+                type: integer
+              bpfMapSizeIPSets:
+                description: BPFMapSizeIPSets sets the size for ipsets map
+                type: integer
+              bpfMapSizeNATAFF:
+                type: integer
+              bpfMapSizeNATBE:
+                description: BPFMapSizeNatBE sets the size for nat back end map. This
+                  is the total number of endpoints. This is mostly more than the size
+                  of the number of services.
+                type: integer
+              bpfMapSizeNATFE:
+                description: BPFMapSizeNatFE sets the size for nat front end map.
+                  This is equal to the number of kubernetes services.
```

Comment:
That is not quite true, nodeports, external IP etc. will create additional entries.